### PR TITLE
fix: disable recover wallet

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

Properly disable the Recover Wallet button.

We are disablIng this button until the recovery flow is complete.

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

N/A

## Testing

N/A

## Screenshots (if applicable)

N/A

### CODE REVIEW

**Review by 0xdef1cafe — APPROVED** [review]

**Review by gomesalexandre — APPROVED** [review]
🍬

**Review by NeOMakinG — APPROVED** [review]

File: src/context/WalletProvider/KeepKey/components/WipedSuccessfully.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export const WipedSuccessfully = () => {
           isFullWidth
           size='lg'
           onClick={handleRecoverWalletPress}
-          disabled={loading && true} // Un-disable when recover wallet is ready
+          disabled={loading || true} // Un-disable when recover wallet is ready
```
Comment (cjthompson) [inline src/context/WalletProvider/KeepKey/components/WipedSuccessfully.tsx:46]: WTF? this will always be true.

Comment (NeOMakinG) [reply src/context/WalletProvider/KeepKey/components/WipedSuccessfully.tsx:46]: I guess the comment is the `@TODO`, actually it should always be disabled until the recover flow will be implemented in the application

Comment (0xApotheosis) [reply src/context/WalletProvider/KeepKey/components/WipedSuccessfully.tsx]: Yeah, it should really be a TODO: https://github.com/shapeshift/web/pull/1490/commits/bfcd85aa6387b109d8da0b3e2adea927dc39462c

**Pull request merged**